app/models/user.rb

```diff
@@ -66,6 +66,8 @@ class User < ApplicationRecord
 
   validate :consents_to_processing, on: :create, unless: ->(user) { user.using_omniauth? || User.current_user.try(:is_admin?) }
 
+  validates :processing_consent, :presence => true
+  
```

Comment:
I think this can be removed now after the changes to `consents_to_processing`.

(Also it will break for existing resources since the `processing_consent` param is only present when the user is first created)